src/App.tsx

```diff
@@ -1,51 +1,73 @@
 import React from "react";
 
 import {IEvent, IChoice} from "./events/core";
-import Player from "./Player";
+import PickFacultyEvent from "./events/PickFacultyEvent";
+import BoomerGregorEvent from "./events/BoomerGregorEvent";
```

Comment:
Can we make a BaseEvent for other Events to inherit from?

Reply:
As we discussed online, there's discussion to refactor how we store events to just use a `.json` file instead of using classes. This will be resolved in a follow up issue.